lib/logstash-docket/util/alias_plugin_util.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+require "yaml"
+require "net/http"
+
+module LogstashDocket
+  module Util
+    ##
+    # A util class defines repetitive logics for aliased plugins.
+    #
+    class AliasPluginUtil
+
+      ALIAS_MAPPINGS_URL = 'https://raw.githubusercontent.com/elastic/logstash/master/logstash-core/src/main/resources/org/logstash/plugins/AliasRegistry.yml'
```

Comment:
`s/master/main/g`?

I think this is working because github is silently handling the primary branch rename, but it is possible that if a `master` branch were ever pushed that aliasing would break.

```suggestion
      ALIAS_MAPPINGS_URL = 'https://raw.githubusercontent.com/elastic/logstash/main/logstash-core/src/main/resources/org/logstash/plugins/AliasRegistry.yml'
```

Reply:
Yeah, it worked because by default it seems rolled back to `main`. Changed to main.